backend/benefit/helsinkibenefit/urls.py

```diff
@@ -11,7 +11,12 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("v1/", include((router.urls, "v1"), namespace="v1")),
-    path("v1/company/<str:business_id>", GetCompanyView.as_view()),
+    path(
+        "v1/company/<str:business_id>", GetCompanyView.as_view()
+    ),  # FIXME: Remove this later
+    path("v1/company/", GetCompanyView.as_view()),
+    path("oidc/", include("oidc.urls")),
```

Comment:
Should this be `include("shared.oidc.urls")`?

Reply:
I think both would work, as "oidc" is also declared in INSTALLED_APPS